### PR TITLE
docs: add from diskto disk methods to docs

### LINKED ISSUE #5178: [DOCS] Add `from_disk/to_disk` methods to docs

### PR DESCRIPTION
**Pull request by sdiazlor**
# Pull Request Template

Closes #5178 

**Type of change**

- Documentation update

**How Has This Been Tested**

**Checklist**

- I added relevant documentation

### CODE REVIEW

**Comment by davidberenstein1957** [comment]
@sdiazlor, looking good. Would you be able to review the line length of code in guides too? sometimes it isn't easy to read because lines are too long. Also, along with the release of the `from_hub` and `to_hub` methods, I think we should try to generalize the headers a bit more and add the different ways to load en save datasets in different tabs.

File: argilla/docs/how_to_guides/dataset.md [hunk 1]
```diff
@@ -490,4 +490,30 @@ client = rg.Argilla(api_url="<api_url>", api_key="<api_key>")
...
+You can save an existing dataset locally by calling the `to_disk` method on the `Dataset` class.
+
+```python
+import argilla as rg
+
+client = rg.Argilla(api_url="<api_url>", api_key="<api_key>")
+
+dataset_to_save = client.datasets(name="my_dataset")
+
+dataset_saved = dataset_to_save.to_disk(path="/path/empty-directory")
```
Comment (dvsrepo) [inline argilla/docs/how_to_guides/dataset.md:506]: What does `to_disk()` returns? I'd use a more precise name for the variable and even add a sentence in the description on top about it. I also think is worth mentioning that we recommend creating an empty directory

**Review by dvsrepo — CHANGES_REQUESTED** [review]
just a small comment

**Review by davidberenstein1957 — COMMENTED** [review]
see earlier comment

**Pull request closed without merging**